src/Features/Core/Portable/InitializeParameter/AbstractInitializeMemberFromParameterCodeRefactoringProviderMemberCreation.cs

```diff
@@ -133,7 +133,7 @@ internal abstract partial class AbstractInitializeMemberFromParameterCodeRefacto
                         default,
                         accessibilityLevel,
                         DeclarationModifiers.ReadOnly,
-                        parameter.Type, uniqueName);
+                        parameter.Type.WithNullability(parameter.NullableAnnotation), uniqueName);
```

Comment:
```suggestion
                        parameter.GetTypeWithAnnotatedNullability(), uniqueName);
```

Reply:
Oh right, that's what @CyrusNajmabadi was complaining about. Fixed.